docs/source/quantization.rst

```diff
@@ -322,10 +322,11 @@ to do the following in addition:
 (Prototype) FX Graph Mode Quantization
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-There are multiple quantization types in post training quantization (weight only, dynamic and static) and the configuration is done through `qconfig_dict` (an argument of the `prepare_fx` function).
+There are multiple quantization types in post training quantization (weight only, dynamic and static) and the configuration is done through `qconfig_mapping` (an argument of the `prepare_fx` function).
 
 API Example::
 
+  from torch.ao.quantization import QConfigMapping
```

Comment:
nit: `torch.quantization` for now to stay consistent with the rest of the document?

we plan to communicate the `ao` part once all the refactors are landed